controllers/idler/idler_controller_test.go

```diff
@@ -338,16 +340,20 @@ func TestEnsureIdlingFailed(t *testing.T) {
 				// then
 				require.EqualError(t, err, fmt.Sprintf("failed to ensure idling 'alex-stage': %s", errMsg))
 				assert.Equal(t, reconcile.Result{}, res)
-				memberoperatortest.AssertThatIdler(t, idler.Name, cl).HasConditions(memberoperatortest.FailedToIdle(errMsg))
+				if first {
+					memberoperatortest.AssertThatIdler(t, idler.Name, cl).HasConditions(memberoperatortest.FailedToIdle(errMsg))
+				} else {
+					memberoperatortest.AssertThatIdler(t, idler.Name, cl).HasConditions(memberoperatortest.FailedToIdle(errMsg), memberoperatortest.IdlerNotificationCreated())
+				}
 			}
 
-			assertCanNotGetObject(&appsv1.Deployment{}, "can't get deployment")
-			assertCanNotGetObject(&appsv1.ReplicaSet{}, "can't get replicaset")
-			assertCanNotGetObject(&appsv1.DaemonSet{}, "can't get daemonset")
-			assertCanNotGetObject(&batchv1.Job{}, "can't get job")
-			assertCanNotGetObject(&appsv1.StatefulSet{}, "can't get statefulset")
-			assertCanNotGetObject(&openshiftappsv1.DeploymentConfig{}, "can't get deploymentconfig")
-			assertCanNotGetObject(&corev1.ReplicationController{}, "can't get replicationcontroller")
+			assertCanNotGetObject(&appsv1.Deployment{}, "can't get deployment", false)
+			assertCanNotGetObject(&appsv1.ReplicaSet{}, "can't get replicaset", false)
+			assertCanNotGetObject(&appsv1.DaemonSet{}, "can't get daemonset", true)
+			assertCanNotGetObject(&batchv1.Job{}, "can't get job", false)
+			assertCanNotGetObject(&appsv1.StatefulSet{}, "can't get statefulset", false)
+			assertCanNotGetObject(&openshiftappsv1.DeploymentConfig{}, "can't get deploymentconfig", false)
+			assertCanNotGetObject(&corev1.ReplicationController{}, "can't get replicationcontroller", false)
```

Comment:
I'm sorry, this test doesn't seem to be the right place for verifying the Notification CR creation - this test is focused on failures and it's not really clear why only for the `DeploymentSet` the notification CR should be created

Reply:
you're right, done in [777b019](https://github.com/codeready-toolchain/member-operator/pull/357/commits/777b019b9dc2a2da36de0b0e2754b8012b10f7d8)